iap/refresh/index.html

```diff
@@ -0,0 +1,13 @@
+<html><body>
```

Comment:
Per the [HTML style guide](https://google.github.io/styleguide/htmlcssguide.html), this should have

```
<!DOCTYPE html>
```

The `<html>` and `<body>` tags are optional and should be removed.

Also, add `<meta charset="utf-8">` and a `<title>` tag.

Reply:
Done.